src/handshake/login.js

```diff
@@ -1,15 +1,12 @@
-const fs = require('fs')
 const JWT = require('jsonwebtoken')
-const DataProvider = require('../../data/provider')
 const { nextUUID } = require('../datatypes/util')
 const { PUBLIC_KEY } = require('./constants')
+const mcData = require('minecraft-data/data').bedrock
```

Comment:
ah the main index.js doesn't work for this anymore ?

Reply:
oops, forgot to expose, fixed in https://github.com/PrismarineJS/node-minecraft-data/pull/115 and https://github.com/PrismarineJS/minecraft-data/pull/465